bedrock/mozorg/urls.py

```diff
@@ -85,12 +85,8 @@
     page("about/policy/transparency/jan-jun-2023/", "mozorg/about/policy/transparency/jan-jun-2023.html"),
     page("contact/", "mozorg/contact/contact-landing.html"),
     page("contact/spaces/", "mozorg/contact/spaces/spaces-landing.html"),
-    page("contact/spaces/beijing/", "mozorg/contact/spaces/beijing.html"),
-    page("contact/spaces/berlin/", "mozorg/contact/spaces/berlin.html"),
     # Paris contact page redirect to contact landing page
     redirect(r"^contact/spaces/paris/$", "mozorg.contact.spaces.spaces-landing", locale_prefix=False),
-    page("contact/spaces/san-francisco/", "mozorg/contact/spaces/san-francisco.html"),
```

Comment:
In theory, now with all `/spaces/*` gone and all ending up at spaces-landing… could the `spaces/paris` redirect L88-89 be removed too, as it's now implicit?

Reply:
oh sorry now I noticed the redirects are added individually:

https://github.com/mozilla/bedrock/blob/564fd97066b99f7a195aac68f07035b56a632f11/bedrock/mozorg/redirects.py#L569-L573

so the `spaces/paris` still needs to be around somewhere; only probably moved from `urls.py` to `redirects.py`…

Reply:
Ah good eye! It seems I added the `spaces/paris` page as a redirect but in the `urls.py` file instead of the `redirects.py` one some months ago, so I just fixed that up now. 👍🏼